[2024-05-06T12:32:57.741+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T12:32:57.762+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:32:45.351407+00:00 [queued]>
[2024-05-06T12:32:57.770+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:32:45.351407+00:00 [queued]>
[2024-05-06T12:32:57.771+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2024-05-06T12:32:57.785+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): insert_to_postgres> on 2024-05-06 12:32:45.351407+00:00
[2024-05-06T12:32:57.792+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2621) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-06T12:32:57.793+0000] {standard_task_runner.py:63} INFO - Started process 2622 to run task
[2024-05-06T12:32:57.794+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'insert_to_postgres', 'manual__2024-05-06T12:32:45.351407+00:00', '--job-id', '298', '--raw', '--subdir', 'DAGS_FOLDER/kachevo.py', '--cfg-path', '/tmp/tmpj1p8pbi9']
[2024-05-06T12:32:57.796+0000] {standard_task_runner.py:91} INFO - Job 298: Subtask insert_to_postgres
[2024-05-06T12:32:57.839+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_operator_dag.insert_to_postgres manual__2024-05-06T12:32:45.351407+00:00 [running]> on host db2363de7199
[2024-05-06T12:32:57.917+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_operator_dag' AIRFLOW_CTX_TASK_ID='insert_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T12:32:45.351407+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T12:32:45.351407+00:00'
[2024-05-06T12:32:57.918+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T12:32:57.947+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-05-06T12:32:58.002+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-06T12:32:58.002+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T12:32:58.014+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=insert_to_postgres, execution_date=20240506T123245, start_date=20240506T123257, end_date=20240506T123258
[2024-05-06T12:32:58.048+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-06T12:32:58.065+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-06T12:32:58.067+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
